_security/authentication-backends/saml.md

```diff
@@ -110,6 +110,21 @@ Name | Description
 `idp.entity_id` | The entity ID of your IdP. Required.
 `sp.entity_id` | The entity ID of the service provider. Required.
 
+## Time disparity compensation for JSON Web Token validation
```

Comment:
I just re-checked SAML and for the `nbf` claim it is using the OpenSearch node's clock to determine this claim: https://github.com/opensearch-project/security/blob/main/src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java#L294-L295

For expiration it is getting the value from the SAML response after successful authentication. I think this is good to leave in here, but SAML is an example where the `nbf` come's from the OpenSearch node's clock and the `exp` comes from the authentication server.